prql-compiler/src/tests/test.rs

```diff
@@ -853,7 +853,7 @@ fn test_window_functions_02() {
         TO_CHAR(co.order_date, '%Y-%m-%d') AS order_day,
         COUNT(DISTINCT co.order_id) AS num_orders,
         COUNT(*) AS num_books,
-        SUM(ol.price) AS total_price
+        COALESCE(SUM(ol.price), 0) AS total_price
```

Comment:
My initial reaction was "is it worth the SQL noise", but actually this does seem quite reasonable.